app/controllers/todo_lists_controller.rb

```diff
@@ -0,0 +1,51 @@
+class TodoListsController < ApplicationController
+    skip_forgery_protection
+    # GET /users/:user_id/todo_lists
+    def index
+      user = User.find(params[:user_id]) 
```

Comment:
add a before action call, to find the user. And set the class variable and access the same in all endpoints.